package.json

```diff
@@ -2,6 +2,9 @@
   "name": "snyk-nodejs-lockfile-parser",
   "description": "Generate a dep tree given a lockfile",
   "main": "dist/lib/index.js",
+  "bin": {
+    "parse": "./bin/index.js"
```

Comment:
I think `"parse"` is probably too generic for a bin that will be installed globally

Reply:
Not sure what a better name might be.. maybe "parse-npm-lockfile"??